packages/standalone-analyzer/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@codecov/standalone-analyzer",
```

Comment:
I feel like `standalone` might be confusing to some users.

could we evaluate some other package name options?

Reply:
Thoughts on `@codecov/asset-analyzer`?

Reply:
`@codecov/asset-analyzer` or `@codecov/bundle-analyzer` feels better to me, but I also suck at names 😅